geomstats/geometry/lie_group.py

```diff
@@ -242,9 +252,12 @@ def group_exp(self, tangent_vec, base_point=None, point_type=None):
         result = gs.cond(
             pred=gs.allclose(base_point, identity),
             true_fn=lambda: self.group_exp_from_identity(
-                tangent_vec, point_type=point_type),
+                tangent_vec, point_type=point_type
+            ),
             false_fn=lambda: self.group_exp_not_from_identity(
-                tangent_vec, base_point, point_type))
+                tangent_vec, base_point, point_type
+            ),
+        )
```

Comment:
We could enforce this parenthesis style: do you feel it would be better?

Reply:
I think this was done by my code formatter, disabled that for now :)